clang/test/CodeGenHLSL/BasicFeatures/MatrixSingleSubscriptDynamicSwizzle.hlsl

```diff
@@ -0,0 +1,10 @@
+// RUN: %clang_cc1 -triple dxil-pc-shadermodel6.7-library -disable-llvm-passes -emit-llvm -finclude-default-header -o - %s | FileCheck %s
+// XFAIL: *
```

Comment:
Do we have an issue number or link you can add here? Or maybe you don't need to add this test at all since there are not CHECK lines.